repo_tests/comment_line_number_citations_test.py

```
@@ -0,0 +1,152 @@
+
+def test_the_baseline_still_describes_real_citations() -> None:
+    """A baseline entry whose citation is gone must be deleted, not carried.
+
+    Otherwise the list grows stale in the same way the citations do, and a
+    future author reads it as a record of what exists.
+    """
+    live = {(rel, cite) for rel, _, cite in _scan()}
+    vanished = sorted(_BASELINE - live)
+    assert not vanished, (
+        "these baseline entries no longer exist — remove them so the list keeps "
+        f"meaning what it says:\n" + "\n".join(f"  {f} -> {c}" for f, c in vanished)
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)